spec/channels/conversation_channel_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe ConversationChannel, type: :channel do
   let(:user)          { create(:user) }
-  let(:other_user)    { create(:user) }
+  let(:other_user)    { create(:user, push_token: ['123a4567-8a9a-12aa-a34a-5aa67a89aaaa']) }
```

Comment:
Do not use hard-coded values. Use faker, or if its necessary (not in this case I think) an env variable.